pytorch/inference/docker/2.2/py3/Dockerfile.cpu

```diff
@@ -118,7 +118,7 @@ RUN curl -L -o ~/miniforge3.sh https://github.com/conda-forge/miniforge/releases
  && /opt/conda/bin/conda install -c conda-forge \
     python=${PYTHON_VERSION} \
     cython \
-    "mkl>=2023.2.0" \
+    "mkl<2024.1.0" \
```

Comment:
What's the reason for this upper pin?

Reply:
old PT binaries are built with old mkl. Without pin, we'd see `undefined symbol: iJIT_NotifyEvent` error. https://github.com/pytorch/pytorch/issues/123097